identifiers/country-id/README

```diff
@@ -18,3 +18,9 @@ OCD ID Types used used:
 *   province: represents a province, or a subdivision of Indonesia's administrative district
 *   special_region: represents a special region, or a subdivision of Indonesia's administrative district with a special status
 *   capital_district: represents a capital district, or a subdivision of Indonesia's administrative district with a special status
+
+Addition of New OCD_ID for both Regions and Dewan_perwakilan_rakyat :
```

Comment:
README should reflect the current state of the files, but it's not necessary to add notes in it of what recent changes were done.